### ISSUE TITLE
Command line labels or Project Files referring to external files

### ISSUE THREAD

**Issue by jaggzh** [audacity/audacity#5129]
### Your idea

I propose two possible options, each with their own use-cases applicable to various users:
1. Providing the ability for an .aup3 file to refer to external audio and labels files
2. _or:_ A command-line option `-labels file.txt`, used like `"audacity file.wav -labels file.txt"`, which will load the audio and add the labels track.

Limitations:
1. While the later (#2) is more convenient for loading them up, without creating project files, it lacks the ability for the tracks to be customized (eg. I convert the audio track to spectrogram, with a 512 width window).
2. While requiring more overhead to create the .aup3 file "template" (referring to external files), it is more powerful.  Nevertheless, I foresee the decision to switch away from external references might be an insurmountable obstacle to getting this implemented, so I think the commandline option might be the easier-implemented ability.

Got any other ideas?

### Problem to be solved

External to Audacity I'm generating audio and label files. It's for neural net training data review and requires doing it over and over.
Loading these into Audacity used to be possible with the loading of external files, I believe, but since aup3 files now include the audio/other data it's no longer possible to load an .aup3 and have the audio pulled in.  (I'm not sure if that was actually possible before).

**Comment by LWinterberg** [comment]
Not being able to load external files is a shortcoming of the current project format; if you use a version <=2.4.2 you'll be able to use that. 

Regarding command line options, try scripting instead - https://manual.audacityteam.org/man/scripting.html It's a bit non-standard but it probably should allow you to do what you want.

**Comment by bwagner** [comment]
@jaggzh, See comments for issue [7171](https://github.com/audacity/audacity/issues/7171#issuecomment-2313539002) for a workaround to import labels.
